pkg/down/down.go

```diff
@@ -53,7 +53,7 @@ func (d *downRunner) deleteCommon(errorChannel chan<- error, kind string, lister
 	}
 }
 
-func (d *downRunner) deleteServices(errorChannel chan<- error) {
+func (d *downRunner) deleteK8sResource(resource string, errorChannel chan<- error) {
```

Comment:
This is an incorrect generalization because a the serviceClient is used to delete pods. E.g. it is listing services but then deleting pods that have the name of a service listed. This is not what the function intended to do.
I remember this one was tricky to deduplicate so just left it, might be better to `// nolint` them.

Reply:
So you're suggesting we re-add the deleteServices & deletePods functions and just // nolint both?